apps/hefty/lib/hefty/trades.ex

```diff
@@ -234,4 +234,50 @@ defmodule Hefty.Trades do
     )
     |> Hefty.Repo.all()
   end
+
+  def get_trades_dates() do
+    query =
+      "SELECT to_char(to_timestamp(buy_time / 1000.0), 'YYYY-MM-DD') AS date " <>
+        "FROM trades GROUP BY to_char(to_timestamp(buy_time / 1000.0), 'YYYY-MM-DD') " <>
+        "ORDER BY to_char(to_timestamp(buy_time / 1000.0), 'YYYY-MM-DD') DESC;"
+
+    result =
+      Ecto.Adapters.SQL.query!(Hefty.Repo, query).rows
+      |> Enum.map(fn elm ->
```

Comment:
Could this be `|> Enum.map(&List.fist/1)` ?

Reply:
Updated. Thx